src/libponyc/codegen/codegen.c

```diff
@@ -688,10 +688,19 @@ static bool init_module(compile_t* c, ast_t* program, pass_opt_t* opt, bool jit)
   const char* version = "ponyc-" PONY_VERSION;
   LLVMMetadataRef fileRef = LLVMDIBuilderCreateFile(c->di, filename,
     strlen(filename), dirname, strlen(dirname));
+#  if PONY_LLVM >= 1101
```

Comment:
we should talk about if we want to delete code for older versions of LLVM. Given that we don't support using other LLVMs, I would be in favor of deleting and only having currently supported version.

I remember us saying we would discuss this eventually. Maybe now is a good to have that "eventually" discussion.

If we did however, I'd want to do that removal as a separate PR.

Reply:
I would be in favour of removing older code, agreed that it should be a separate PR.